edge/pkg/devicetwin/dtcommon/common.go

```diff
@@ -49,6 +49,8 @@ const (
 	DeviceETUpdatedSuffix = "/updated"
 	// DeviceETStateUpdateSuffix the topic suffix for device state update event
 	DeviceETStateUpdateSuffix = "/state/update"
+	// DeviceETStateUpdateResultSuffix the topic suffix for device state update result event
+	DeviceETStateUpdateResultSuffix = "/state/update/result"
```

Comment:
there're some codes that also can be replaced by this new topic in edge/test/integration/device/device_test.go, we can modify those also.